app/controllers/rankings_controller.rb

```diff
@@ -57,32 +57,40 @@ def end
 
   end
 
-  # POST /rankings
-  # POST /rankings.json
-  def create
-    @times.each do |time|
-      unless params.keys.include? time.id.to_s or time.is_date
-        flash[:error] = "Error: please fill in an option for each time."
-        redirect_to edit_project_participant_ranking_path(:secret_id => @participant.secret_id) and return
+  def each_time_in_params?
+    @times.where(:is_date => false).each do |time|
```

Comment:
Thanks for refactoring this, but we don't need it anymore lol because the slider guarantees a default value.